rusqlite_migration/src/loader.rs

```diff
@@ -0,0 +1,112 @@
+use std::collections::BTreeMap;
+
+use crate::{Error, Result, M};
+use include_dir::{Dir, DirEntry};
+use log::trace;
+
+#[derive(Debug, Clone)]
+struct MigrationFile {
+    name: &'static str,
```

Comment:
that information could be kept as a new `comment` field in M, visible with Debug, it would help identifying which file create a particular problem. And then a new method could be added to M to define those comments in the “normal” way as well.

Reply:
The whole filename or just the part between the `id-` and `-up/down.sql`?

Reply:
maybe just the `id-and-the-part-in-the-middle`, without the `up/down.sql`. My thinking is that it’ll help when debugging to find the file that created a particular migration.